addHack.js

```diff
@@ -19,13 +19,23 @@ export const add = async (event, context) => {
     });
     conn.model(
       "Hack",
-      new mongoose.Schema({ title: String, description: String, goal: String, team: Array })
+      new mongoose.Schema({
+        title: { type: String, default: "" },
+        description: { type: String, default: "" },
+        goal: { type: String, default: "" },
+        team: { type: Array, default: [] }
```

Comment:
Can we not add the team name when creating the hack in the first place?

Reply:
on the front end at the moment when creating the hack only allows title, description and goal